Readme.md

```diff
@@ -37,6 +37,12 @@ make integration
 - [ ] Key
 - [ ] Protocol Mappers
 - [ ] Realms Admin
+  - [x] Get realm
+  - [~] Import realm
```

Comment:
Doesn't look like GH markdown supports partial checkboxes.

Reply:
Since import realm is not fully featured, I'll leave it empty.